app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,25 @@ def show
     @post = @user.posts.find(params[:id])
     @comments = @post.comments.all
   end
+
+  def new
+    @post = Post.new
+    @current_user = current_user
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:author_id, :title, :text, :comments_counter, :likes_counter))
+
+    respond_to do |format|
+      format.html do
+        redirect_to user_post_path(@post.user.id, @post.id) if @post.save
+      end
+    end
+  end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
```

Comment:
- [ ] `post_params` never used here. Please make sure to use the correct params while creating a post.

Reply:
Done ✅